src/Core_Command.php

```diff
@@ -673,7 +676,7 @@ private function _multisite_convert( $assoc_args ) {
 \$base = '{$assoc_args['base']}';
 define( 'DOMAIN_CURRENT_SITE', '{$domain}' );
 define( 'PATH_CURRENT_SITE', '{$assoc_args['base']}' );
-define( 'SITE_ID_CURRENT_SITE', 1 );
+define( 'SITE_ID_CURRENT_SITE', $site_id );
```

Comment:
CS: Use curly braces around variable substitution
```suggestion
define( 'SITE_ID_CURRENT_SITE', {$site_id} );
```